### PR TITLE
Feature/beautify medium

### PR DESCRIPTION
**Pull request by TuxedoFish**
## What does this PR do?
Makes text more visible on medium article cards by adding a black overlay background. Also adds a zoom and brighten animation on hover.

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?
- [ ] Are docs written for this PR? (if applicable)

## 🚨 Is this a breaking change for API clients?
No

## :squirrel: Deploy notes
npm

## Anything else
animated

unanimated

### CODE REVIEW

File: frontend/src/sass/common/uclapi.scss [hunk 1]
```diff
@@ -258,6 +258,17 @@ a:hover { color: $color-grey-light; }
   overflow: hidden;
 }
 
+.uclapi-card img {
+    -webkit-filter: brightness(75%);
+    -webkit-transition: .5s linear;
```
[the remark this answers is not in the document]
Comment (TuxedoFish) [reply frontend/src/sass/common/uclapi.scss]: Vendor prefixes~

File: frontend/src/sass/common/uclapi.scss [hunk 2]
```diff
@@ -258,6 +258,17 @@ a:hover { color: $color-grey-light; }
   overflow: hidden;
 }
 
+.uclapi-card img {
+    -webkit-filter: brightness(75%);
```
[the remark this answers is not in the document]
Comment (TuxedoFish) [reply frontend/src/sass/common/uclapi.scss]: Vendor prefixes~

[the remark this answers is not in the document]
Comment (hueyy) [reply frontend/src/sass/common/uclapi.scss]: @TuxedoFish should we add a generic `filter: ` for non-webkit browsers, e.g. firefox?

File: frontend/src/sass/common/uclapi.scss [hunk 3]
```diff
@@ -256,6 +256,17 @@ a:hover { color: $color-grey-light; }
   margin: 2%;
   min-height: fit-content;
   overflow: hidden;
+
+  img {
+    -webkit-filter: brightness(75%);
```
Comment (hueyy) [inline frontend/src/sass/common/uclapi.scss]: @TuxedoFish should we add a generic filter: for non-webkit browsers, e.g. firefox?

Comment (TuxedoFish) [reply frontend/src/sass/common/uclapi.scss]: Yeh, there is no need for any of them to use webkit have taken that out!

**Pull request merged**